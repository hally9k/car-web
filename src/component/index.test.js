import { testStatechart } from 'react-automata'
import { Root, statechart } from './index.jsx'

test('it works', () => {
	testStatechart({ statechart }, Root)
})
