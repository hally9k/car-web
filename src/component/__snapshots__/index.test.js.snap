// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works: green 1`] = `
<div
  className="app"
>
  <div
    className="traffic-light"
  >
    <div
      className=""
    />
    <div
      className=""
    />
    <div
      className="lit"
    />
  </div>
  <div
    className="cross-button"
    onClick={[Function]}
  />
</div>
`;

exports[`it works: orange 1`] = `
<div
  className="app"
>
  <div
    className="traffic-light"
  >
    <div
      className=""
    />
    <div
      className="lit flashing"
    />
    <div
      className=""
    />
  </div>
  <div
    className="cross-button"
    onClick={[Function]}
  />
</div>
`;

exports[`it works: red 1`] = `
<div
  className="app"
>
  <div
    className="traffic-light"
  >
    <div
      className="lit"
    />
    <div
      className=""
    />
    <div
      className=""
    />
  </div>
  <div
    className="cross-button"
    onClick={[Function]}
  />
</div>
`;
